.github/workflows/graalpy-install-package.yml

```
@@ -0,0 +1,40 @@
+name: graalpy-install-package
+on:
+  push:
+    paths:
+      - 'graalpy-install-package/**'
+      - '.github/workflows/graalpy-install-package.yml'
+  # pull_request: (requires EE, which is unavailable in PRs)
```

That's not true anymore. Either remove this or use this:
```suggestion
  pull_request:
    paths:
      - 'graalpy-install-package/**'
      - '.github/workflows/graalpy-install-package.yml'
```